api/src/main/java/com/code4ro/nextdoor/emergency/contact/service/impl/EmergencyContactServiceImpl.java

```diff
@@ -0,0 +1,93 @@
+package com.code4ro.nextdoor.emergency.contact.service.impl;
+
+import com.code4ro.nextdoor.emergency.contact.dto.EmergencyContactDto;
+import com.code4ro.nextdoor.emergency.contact.entity.EmergencyContact;
+import com.code4ro.nextdoor.emergency.contact.repository.EmergencyContactRepository;
+import com.code4ro.nextdoor.emergency.contact.service.EmergencyContactService;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EmergencyContactServiceImpl implements EmergencyContactService {
+
+    private final EmergencyContactRepository emergencyContactRepository;
+
+    public EmergencyContactServiceImpl(EmergencyContactRepository emergencyContactRepository) {
```

Comment:
Sa adaugi anotarea @ Inject

Reply:
Citind commentul ma gandeam ca oare cum am uitat si totusi mergea..am descoperit asta: https://docs.spring.io/spring-boot/docs/2.0.3.RELEASE/reference/html/using-boot-spring-beans-and-dependency-injection.html

O sa adaug ca in cazul in care se mai pun parametri sa nu se mai modifice.